persistence/src/test/scala/hmda/persistence/institutions/FilingPersistenceSpec.scala

```diff
@@ -3,48 +3,67 @@ package hmda.persistence.institutions
 import akka.testkit.TestProbe
 import hmda.model.fi._
 import hmda.persistence.messages.CommonMessages.GetState
-import hmda.persistence.demo.DemoData
 import hmda.persistence.institutions.FilingPersistence._
 import hmda.persistence.model.ActorSpec
 
 class FilingPersistenceSpec extends ActorSpec {
 
   val filingsActor = createFilings("12345", system)
 
+  val sample1 = Filing("2016", "12345", Cancelled, filingRequired = true, 1483287071000L, 1514736671000L)
+  val sample2 = Filing("2017", "12345", NotStarted, filingRequired = true, 0L, 0L)
+
   val probe = TestProbe()
 
-  "Filings" must {
-    "be created and read back" in {
-      val filings = DemoData.testFilings
+  "CreateFiling" must {
+    "create a filing" in {
+      val filings = Seq(sample1, sample2)
       for (filing <- filings) {
         probe.send(filingsActor, CreateFiling(filing))
         probe.expectMsg(Some(filing))
       }
       probe.send(filingsActor, GetState)
       probe.expectMsg(filings.reverse)
     }
-    "be able to change their status" in {
-      val filing = DemoData.testFilings.head
-      val modified = filing.copy(status = Cancelled)
-      probe.send(filingsActor, UpdateFilingStatus(modified))
-      probe.expectMsg(Some(modified))
-      probe.send(filingsActor, GetFilingByPeriod(filing.period))
-      probe.expectMsg(modified)
-    }
-
-    "return None when persisting a filing that already exists" in {
-      val f1 = Filing("2016", "12345", Completed, true, 1483287071000L, 1514736671000L)
+    "return None when persisting a filing period that already exists" in {
+      val f1 = Filing("2018", "12345", Completed, true, 1483287071000L, 1514736671000L)
```

Comment:
We shouldn't need this first section of the test, right?  It's only supposed to make sure `None` is returned for the 2016 filing.

Reply:
👍  Gave that spec a more accurate name in fb10cde